packages/atomic/src/components/common/generated-answer/generated-answer-common.tsx

```diff
@@ -193,6 +193,41 @@ export class GeneratedAnswerCommon {
       });
   }
 
+  private renderFeedbackAndCopyButtons() {
+    if (this.props.getGeneratedAnswerState()?.isStreaming) {
+      return null;
+    }
+    return (
+      <div class="feedback-buttons flex h-9 absolute top-6 right-20 shrink-0 gap-2">
+        <FeedbackButton
+          title={this.props.getBindings().i18n.t('this-answer-was-helpful')}
```

Comment:
Small refactor opportunity while touching these function: Destructure to remove all those repetitive and verbose `this.props.getBindings().[...]`

Something like:

```typescript
const {
      getBindings,
      getGeneratedAnswerState,
      getCopyError,
      getCopied,
      getGeneratedAnswer,
    } = this.props;
    const {i18n} = getBindings();
    const {
      isStreaming,
      liked,
      disliked,
      answer,
      citations,
      responseFormat: {answerStyle},
      error,
    } = getGeneratedAnswerState() ?? {};
    const {like, dislike} = getGeneratedAnswer() ?? {};
```

Reply:
@olamothe good point I am doing the refactor of this file in the show more button PR, I will include this in that PR as a lot of refactoring is going on there, regarding code reviews suggestions !